ITEM.tsv

```diff
@@ -22,11 +22,11 @@ ITEM_20150317_000021	$Recommended for those who desire a heavier and stronger sw
 ITEM_20150317_000022	$Sabre 
 ITEM_20150317_000023	$Primary weapon of the Cavalry, optimized for slashing. The guard on the hilt helps protect the user's hand. 
 ITEM_20150317_000024	$Shamshir 
-ITEM_20150317_000025	$The large curved crescent edge allows for a powerful slash. 
+ITEM_20150317_000025	$A blade that has a long, curved crescent. It dishes out heavy slashes.
 ITEM_20150317_000026	$Scimitar 
-ITEM_20150317_000027	$Its heavy weight is compensated by its very powerful slash. 
+ITEM_20150317_000027	$Its heavy weight is compensated by its very powerful slashes. 
 ITEM_20150317_000028	$Kris 
-ITEM_20150317_000029	$The twisted edge amplifies the damage it inflicts. A sword that is both practical and decorative. 
+ITEM_20150317_000029	$A snakelike sword that is both practical and decorative. 
```

Comment:
As for this line, the original line is also closer with the Korean text.

We can stick with snakelike vs twisted, but it does say the edge of the sword magnifies/amplifies the damage it inflicts.